src/scripts/init.ts

```diff
@@ -219,8 +244,7 @@ async function getAccountInfo(config: CliInput) {
     return {
         mqttEndpoint: accountInfo.body.mqttEndpoint,
         mqttTopicPrefix: accountInfo.body.mqttTopicPrefix,
-        tenantId,
-        accountDeviceClientId: `account-${tenantId}`
```

Comment:
This is the bug of the hardcoded account device ID, replaced withe grabbing the team device ID.